test/unit/core/store/middleware.ts

```diff
@@ -218,4 +224,61 @@ suite('Middleware', ({ expect, spy, stub }) => {
       expect(thunk).to.be.calledWithExactly(state);
     });
   });
+
+  describe('personalizationAnalyzer()', () => {
```

Comment:
try to match up the object types here `conf` should be an `object`, `state` should be an `object`

Reply:
can also probably be reused across multiple `it()`s

Reply:
👍

Reply:
Done.